neurodamus/stimulus_manager.py

```diff
@@ -422,16 +438,15 @@ def parse_check_stim_parameters(self, stim_info: dict):
         if self.sd <= 0:
             raise Exception("%s stdev must be positive" % self.__class__.__name__)
 
-        # coefficient of variation of shot amplitudes [1]
-        cv = float(stim_info["AmpCV"])
-        if cv <= 0:
-            raise Exception("%s amplitude CV must be positive" % self.__class__.__name__)
-        self.cv_square = cv * cv
+        # relative skewness of signal as a [0,1] fraction [1]
+        self.rel_skew = float(stim_info.get("RelativeSkew", 0.5))
```

Comment:
Hi @seirios , have you added the corresponding new parameter in the SONATA spec?

Reply:
I have now :)